### ISSUE TITLE
We need a WSL environment in order to test the compatibility

### ISSUE THREAD

**Issue by rekire** [flutter/flutter#142856]
### Type of Request

feature request

### Infrastructure Environment

Not sure to be honest.

### What is happening?

I would like to build the web application from the Windows subsystem for Linux (WSL) see #142619. There fore I created a patch #142803 which cannot be currently tested. In discord I get the feedback that a new CI runner would be required for that case. General speaking when you already have a working Windows CI this should not be very hard, you just need to install the WSL.

A related bug is also #93164 without this CI runner that bug never can be fixed.

### Steps to reproduce

Basically run Windows binaries from bash this is currently not possible

### Expected results

I would like to check the behavior of the flutter SDK when executed on WSL

**Comment by rekire** [comment]
Is that not the other direction? @eliasyishak

Since no pull requests should be accepted without tests this issue blocks my PR.

**Comment by andrewkolos** [comment]
Closing since support for WSL is not planned.
